core-services/template-deprecation/_allowlist.yaml

```diff
@@ -3025,6 +3025,10 @@ templates:
     template_name: prow-job-cluster-launch-installer-remote-libvirt-e2e
     unknown_blocker:
       description: unknown
+      jobs:
+        release-openshift-origin-installer-e2e-remote-libvirt-ppc64le-4.7:
+          generated: false
+          kind: release
```

Comment:
Please add this to the list of jobs blocked by DPTP-1735 above

Reply:
Done